app/screens/home/DetailsScreen.tsx

```diff
@@ -66,7 +66,7 @@ const DetailsScreen = ({route, navigation}: any): any => {
   const {headerColor, fromScreen, backgroundColor,detailData, listCategoryArray, selectedChildActivitiesData, currentSelectedChildId} = route.params;
   console.log(detailData,"..detailData...",fromScreen,"...fromScreen..");
   let newHeaderColor,newBackgroundColor;
-  if(fromScreen === 'Activities' || fromScreen === 'MileStoneActivity' || fromScreen === 'HomeAct' || fromScreen === 'FavActivities')
+  if(fromScreen === 'Activities' || fromScreen === "ActivitiesFirebase" || fromScreen === 'MileStoneActivity' || fromScreen === 'HomeAct' || fromScreen === 'FavActivities')
```

Comment:
I would rename it to FirebaseActivities to be consistent with FavActivities, MileStoneActivity, HomeAct.